### PR TITLE
Project popup window

### PR DESCRIPTION
**Pull request by FevenSeyfu**
Milestones,

- Added a popup window to display project detail
- Updated the work section to render dynamically through javascript
- styled work cards for both desktop and mobile version

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -44,4 +56,199 @@ hamburgerBtn.addEventListener('click', displayMobileMenu);
...
+}
+if (desktopScreen.matches) {
+  const workCard = document.querySelectorAll('.work-card');
+  workCard.forEach((card) => {
+    card.addEventListener('click', (e) => {
+      e.preventDefault();
+      const id = e.target.parentNode.getAttribute('data-id');
+      openPopUpDesktop(id);
+    });
+  });
```
Comment (Favourezeugwa) [inline script.js]: - Great Job adding the modal popup to your portfolio, but I guess there may be a bug causing all project cards to disappear when the `close` button is clicked on. 
Here is what I mean 👇🏽 

| After clicking the close button on the popup  |
| :---:   | 
|Screenshot 2023-01-05 at 18 05 18  |
⚠️ Kindly note that this happens on both mobile and desktop screens. 

- Also, I think some content of the modal popup on mobile isn't being displayed. Can we try to stick to Figma as much as possible? 🤗 

This is what I observed from my end 👇🏽 

| Your mobile popup  | That of Figma |
| :---:   | :---: | 
| Screenshot 2023-01-05 at 18 04 41 |Screenshot 2023-01-05 at 18 25 09  |

**Review by Favourezeugwa — CHANGES_REQUESTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @FevenSeyfu,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-brianna


## Highlights 💯 
- No Linter errors ✔️
- Descriptive Pull request summary✔️
- Good commit messages ✔️
- Professional Readme file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (FevenSeyfu) [reply script.js]: Thank you for your feedback. As per the comments I have made the following changes,

-  fixed the bug that hides the work cards when the popup window gets closed
- I have styled the mobile version of the popup window

File: script.js [hunk 2]
```diff
@@ -44,4 +56,199 @@ hamburgerBtn.addEventListener('click', displayMobileMenu);
...
+    featured_image: 'images/featured-images/project-img-1.png',
+    technologies: {
+      0: 'html',
+      1: 'Bootstrap',
+      2: 'Ruby on rails',
+    },
+    linkTo_live: '#',
+    linkTo_source: '#',
+  },
+  {
```
Comment (iambenkis) [inline script.js:77]: - [ ] [**OPTIONAL**] When you want to see a project you are supposed to see the same description, the same picture, the same project title. Kindle make sure you correct that ! :+1: 

### Before click

Screenshot from 2023-01-06 20-03-29


### Popup After click

Screenshot from 2023-01-06 20-03-46

**Review by iambenkis — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @FevenSeyfu 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters' error.
:heavy_check_mark: Details PR title and description. 

## Optional 
**_Every comment with the [**OPTIONAL**] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@iambenkis** in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by FevenSeyfu** [comment]
Completed Popup window

**Pull request merged**